src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const listItems = document.querySelectorAll('.tree li');
+
+  listItems.forEach((item) => {
+    const child = item.querySelector('ul');
+
+    if (!child) {
+      return;
+    }
+
+    const span = document.createElement('span');
+
+    span.textContent = item.firstChild.textContent.trim();
+    item.firstChild.replaceWith(span);
+
+    child.style.display = 'block';
+
+    span.addEventListener('click', function (e) {
+      e.stopPropagation();
+
+      if (child.style.display === 'none') {
+        child.style.display = 'block';
+      } else {
+        child.style.display = 'none';
+      }
+    });
```

Comment:
This click event listener toggles the display of the `child` element. However, instead of manually checking the display property and setting it, you can use the `classList.toggle` method to add or remove a class that handles the display. This can make the code cleaner and more efficient.